src/Symfony/Component/Form/composer.json

```diff
@@ -33,7 +33,7 @@
         "symfony/dependency-injection": "^4.4|^5.0|^6.0",
         "symfony/expression-language": "^4.4|^5.0|^6.0",
         "symfony/config": "^4.4|^5.0|^6.0",
-        "symfony/console": "^4.4|^5.0|^6.0",
+        "symfony/console": "^5.4|^6.0",
```

Comment:
the conflict line should also be updated

Reply:
done